Frends.MicrosoftDynamics365.Request/Frends.MicrosoftDynamics365.Request/Frends.MicrosoftDynamics365.Request.cs

```diff
@@ -94,7 +94,7 @@ private static async Task<JToken> ExecuteRequestAsync(Input input, Options optio
         }
 
         var jsonResponse = await response.Content.ReadAsStringAsync(cancellationToken);
-        var contacts = JToken.Parse(jsonResponse);
+        var contacts = !string.IsNullOrWhiteSpace(jsonResponse) ? JToken.Parse(jsonResponse) : JToken.Parse("{\r\n\"message\": \"Success\"\r\n}");
```

Comment:
Instead of "message" Success lets not return anything really, since that is the case (no need to have a pretend-value here). So a null would be fine. And we have a boolean Success property on result already which would indicate success anyway.